README.md

```diff
@@ -83,7 +83,7 @@ You can ask questions by raising an [issue](https://github.com/EddieHubCommunity
 - **Add your name to the section that matches your Initial in [this list](https://github.com/EddieHubCommunity/hacktoberfest-practice#hacktoberfest-community). Then, add your name in alphabetical order. Then save your changes.**
 
 - For example
-  `- [Full Name](https://github.com/your-username)`
+  `- [amarjeet kumar](https://github.com/amarjeetk06)`
```

Comment:
Don't change this line, you only head to the section with your alphabet which is section "A" and then put your name in alphabetical order